e2e/test_analyze/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.17.9-alpine AS builder
```

Comment:
what is this `test_analyze` folder for?

Reply:
vulnerable code source and image to scan

Reply:
maybe need to add a readme or some comment to understand what is it for